crates/aptos/src/account/create.rs

```diff
@@ -99,26 +104,60 @@ impl CreateAccount {
         client.submit_and_wait(&transaction).await
     }
 
-    async fn execute_inner(self) -> Result<String, Error> {
+    async fn create_account_with_faucet(self, address: AccountAddress) -> Result<String, Error> {
+        let response = reqwest::Client::new()
+            // TODO: Currently, we are just using mint 0 to create an account using the faucet
+            // We should make a faucet endpoint for creating an account
+            .post(format!(
+                "{}/mint?amount={}&auth_key={}",
+                "https://faucet.devnet.aptoslabs.com", "0", address
+            ))
+            .send()
+            .await?;
+        if response.status() == 200 {
+            Ok(response.status().to_string())
+        } else {
+            Err(Error::new(CommonError::UnexpectedError(format!(
+                "Faucet issue: {}",
+                response.status()
+            ))))
+        }
```

Comment:
btw, there is a `FaucetClient` so you can probably just say `FaucetClient::new(account: AccountAddress)`

Reply:
I tried it, but it's blocking which doesn't play nice with command line. Would have to refactor FaucetClient a little I think.